apps/mobile/app/(import-accounts)/_layout.tsx

```diff
@@ -0,0 +1,23 @@
+import { Stack } from 'expo-router'
+import { HeaderBackButton } from '@react-navigation/elements'
+
+export default function ImportAccountsLayout() {
+  return (
+    <Stack
+      screenOptions={({ navigation }) => ({
+        headerBackButtonDisplayMode: 'minimal',
+        headerShadowVisible: false,
+        headerLeft: (props) => (
+          <HeaderBackButton {...props} testID={'go-back'} onPress={navigation.goBack} displayMode={'minimal'} />
```

Comment:
add this property in the HeaderBackButton in order to have the arrow following the same standard we have in the signers screens?  `tintColor={theme.primary.get()}`

Reply:
I think that we are way out of the current designs. We have a different icon there. But I moved the whole screenOptions to a centeral function since we had copy pasted the above lines in 3 different files.